lib/Ast_to_Mir.ml

```diff
@@ -3,11 +3,36 @@
 open Core_kernel
 open Mir
 
+let trans_op (op : Ast.operator) =
+  match op with
+  | Ast.Plus -> Plus
+  | Ast.Minus -> Minus
+  | Ast.Times -> Times
+  | Ast.Divide -> Divide
+  | Ast.Modulo -> Modulo
+  | Ast.Or -> Or
+  | Ast.And -> And
+  | Ast.Equals -> Equals
+  | Ast.NEquals -> NEquals
+  | Ast.Less -> Less
+  | Ast.Leq -> Leq
+  | Ast.Greater -> Greater
+  | Ast.Geq -> Geq
+  | _ ->
+      raise_s [%message "Should never try to translate " (op : Ast.operator)]
```

Comment:
Is that because they should be represented with function calls instead?

Reply:
Yep. I can add that to the message.